sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayoutController.java

```diff
@@ -185,8 +185,8 @@ private void initSkinViews(Application app, OoyalaSkinLayout l, SkinOptions skin
     rootView = new ReactRootView(l.getContext());
     _reactInstanceManager = ReactInstanceManager.builder()
         .setApplication(app)
-        .setBundleAssetName(skinOptions.getBundleAssetName())
         .setJSBundleFile("assets://index.android.jsbundle")
+        .setBundleAssetName(skinOptions.getBundleAssetName())
```

Comment:
What point left here .setJSBundleFile("assets://index.android.jsbundle") call? If you are fearing  skinOptions.getBundleAssetName() returned something wrong i suggest modify getBundleAssetName() body:
```java
  public String getBundleAssetName() {
    if (bundleAssetName == null) {
      return "index.android.jsbundle";
    }
    return bundleAssetName;
  }
```

Reply:
Removed that line.  `index.android.jsbundle`  is default value for `bundleAssetName`, so it should work fine without this modification